### ISSUE TITLE
Do not generate extra MANIFEST headers by default

### ISSUE THREAD

**Issue by mickaelistria** [eclipse-tycho/tycho#1999]
Eclipse encourages reusing as much as possible existing artifacts. This is usually made simpler by avoiding every build-infra-dependent behavior.
https://github.com/eclipse-m2e/m2e-core/issues/1211 shows that MANIFEST headers such as Build-Jdk-Spec are causing more trouble than help in Eclipse plugins. I suggest all information that are strongly depending on build infra or build time should be removed as there is a better strategy already in place.
From ManifestComparator.IGNORED_KEYS, I suggest
```
           // these keys are added by plexus archiver
            new Name("Archiver-Version"), //
            new Name("Created-By"), //
            new Name("Build-Jdk"), //
            new Name("Built-By"), //
            new Name("Build-Jdk-Spec"),
```
shouldn't be added to bundle MANIFEST by default.

**Comment by laeubi** [comment]
I think the comment

>  // these keys are added by plexus archiver

tells that not Tycho is adding those.... this si configured through https://maven.apache.org/shared/maven-archiver/

**Comment by mickaelistria** [comment]
> tells that not Tycho is adding those

Yes, but Tycho can probably configure its usage of plexus to avoid generating those.
